run-k8s-external-storage-e2e.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+#
+# Download the Kubernetes test suite, extract it and run the external-storage suite.
+#
+# Requirements:
+# - KUBE_VERSION needs to be set in the environment (format: "v1.18.5")
+
+# exit on failure
+set -e
```

Comment:
can you add a link to how this is configured? it might be useful if we want to add more cases

Reply:
yes, I really need to do that. Documentation is very limited though - https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external is all I have